src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -31,21 +31,15 @@ object ProtocPlugin extends AutoPlugin {
   }
 
   // internal key for detect change options
-  private[this] val arguments = TaskKey[Arguments]("protoc-arguments")
+  private[this] val arguments = TaskKey[Arguments]("protocArguments")
```

Comment:
All the other keys above are with dashes, so it won't make sense just to change this. Can we stay with the discouraged way (for the sake of consistency) and change all at once in a future major release?